packages/api/package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "attw": "attw --pack --format table-flipped",
     "build": "tsc",
-    "debug:bin": "node -r ts-node/register src/bin.ts",
+    "debug:bin": "ts-node --esm src/bin.ts",
```

Comment:
`ts-node` doesn't support ESM by default and requires this flag otherwise this funky error is thrown:

![Screen Shot 2023-10-16 at 11 28 08 AM](https://github.com/readmeio/api/assets/33762/52253a84-ddd1-4ea4-b7b3-2650829a79af)

Reply:
I'm still seeing that error, even with the `--esm` flag 🤔 I tried `npm ci && npm run build` beforehand to no avail:

![CleanShot 2023-10-16 at 14 08 15@2x](https://github.com/readmeio/api/assets/8854718/ad8779b9-61f6-4d91-b122-1d58875b44ff)

Reply:
Fixed in https://github.com/readmeio/api/pull/754/commits/9fd037041d0d33b15f7f0fdcc5ab581950c3b94a